src/Sylius/Bundle/AddressingBundle/spec/Twig/CountryNameExtensionSpec.php

```diff
@@ -39,4 +40,16 @@ function it_translates_country_code_to_name_according_to_locale(): void
     {
         $this->translateCountryIsoCode('IE', 'es')->shouldReturn('Irlanda');
     }
+
+    function it_fallbacks_to_country_code_when_there_is_no_translation(): void
+    {
+        $country = 'country_code_without_translation';
+        $this->translateCountryIsoCode($country)->shouldReturn($country);
+    }
+
+    function it_fallbacks_to_an_empty_string_when_there_is_no_code(): void
+    {
+        $this->translateCountryIsoCode(new Country())->shouldReturn('');
```

Comment:
This should be a `CountryInterface` mock with `$country->getCode()->willReturn('country_code_without_translation')` result stubbed

Reply:
You mean `$country->getCode()->willReturn(null)`?